simapp/app_config.go

```diff
@@ -12,6 +12,7 @@ import (
 	authmodulev1 "cosmossdk.io/api/cosmos/auth/module/v1"
 	authzmodulev1 "cosmossdk.io/api/cosmos/authz/module/v1"
 	bankmodulev1 "cosmossdk.io/api/cosmos/bank/module/v1"
+	benchmarkmodulev1 "cosmossdk.io/api/cosmos/benchmark/module/v1"
```

Comment:
```suggestion
        // THIS IS A TESTING ONLY MODULE
	benchmarkmodulev1 "cosmossdk.io/api/cosmos/benchmark/module/v1"
```

should we do this or have a build flag that injects it?

Reply:
I am opting for a build flag to compose the module config in.  That way it's disabled by default.